.eslintrc.js

```diff
@@ -0,0 +1,76 @@
+module.exports = {
+    extends: ['airbnb', 'prettier', 'prettier/flowtype', 'prettier/react'],
```

Comment:
Isn't there any other solution? If prettier is getting real popular - which it lookes like - isn't there a popular eslint-config to match it?

This sounds like duck-taping things together, and also, looking at the config file starts to get complicated.

Eslint is just a list of rules, and a list of plugins. Should be easier